helpers/python.ts

```diff
@@ -268,6 +269,29 @@ export const installPythonTemplate = async ({
         cwd: path.join(compPath, "loaders", "python", loaderFolder),
       });
     }
+
+    // Generate loader configs
+    if (dataSource?.type === "web") {
+      const config = dataSource.config as WebSourceConfig[];
+      const webLoaderConfig = config.map((c) => {
+        return {
+          base_url: c.baseUrl,
+          prefix: c.prefix || c.baseUrl,
+          depth: c.depth || 1,
+        };
+      });
+      const loaderConfigPath = path.join(root, "loaders.json");
```

Comment:
better move it to config and also move the tools config
```suggestion
      const loaderConfigPath = path.join(root, "config/loaders.json");
```

Reply:
this means we also need one two sentences in the README that that the loaders and tools can be configured there